templates/vue/src/components/tyde/activities/CircleOfSupport/ConnectionsTab/AddConnectionForm.vue

```diff
@@ -0,0 +1,289 @@
+<template>
+  <div>
+    <add-community-form
+      v-show="showCommunityForm"
+      @back="showCommunityForm = false"
+      @add-community="handleAddCommunity"
+    />
+    <form v-show="!showCommunityForm" style="height: 100%" @submit.stop.prevent>
+      <div class="connection" style="flex: 1">
+        <input
+          v-model="connection.name"
+          name="connection name"
+          class="connection-name"
+          type="text"
+          placeholder="connection"
+          @blur="isInputTouched = true"
+        />
+        <b-form-invalid-feedback :state="isNameValid">
+          Please enter a name for your connection.
+        </b-form-invalid-feedback>
+        <div id="emoji-picker" style="position: relative">
+          <button class="preview" @click="showPicker = !showPicker">
+            {{ connection.avatar }}
+          </button>
+          <div v-show="showPicker" class="picker" data-qa="emoji-picker">
+            <v-emoji-picker @select="handleEmojiSelect" />
+          </div>
+        </div>
+        <div class="controls">
+          <button @click="$emit('back')">Cancel</button>
+          <button class="submit" @click="submitConnection">
+            {{ submitLabel }}
+          </button>
+        </div>
+      </div>
+      <div class="community" style="flex: 2">
+        <h1 class="community-title">Which communities do this person belong to?</h1>
```

Comment:
This should read "Which communities does this person belong to?"

Reply:
```suggestion
        <h1 class="community-title">Which communities does this person belong to?</h1>
```